client/js/components/user/CustomerSettings/CustomerSettingsSignLanguageVideo.vue

```diff
@@ -121,17 +120,25 @@ export default {
           title: ''
         }
       }
-    }
+    },
+    signLanguageOverviewDescription: {
+      required: false,
+      type: String,
+      default: ''
+    },
   },
 
   data () {
     return {
-      isBusy: false,
```

Comment:
see above loading state and following

Reply:
because the functionality is changed here due to the other buttons, all of them in the customersettings has no busy data

Reply:
I think it is generally a good idea to have a visual indication for users that something is happening in the background. Would it be possible to re-add those loading states?

Reply:
so you mean to add it for all buttons?

Reply:
Only if the button is triggering an api call which results in awaiting data so we can indicate the user that data will be fetched.

Reply:
thanks for the advice. done here https://github.com/demos-europe/demosplan-core/pull/2212/commits/e09ca1f649d4ecf10ba9f280eba551c7c41900ed